### ISSUE TITLE
precision @ 1 = 0.0 resulting for Inception - ImageNet model eval

### ISSUE THREAD

**Issue by julialintern** [tensorflow/models#190]
I'm running Inception-V3 on ImageNet data  (as per https://github.com/tensorflow/models/tree/master/inception README); 

I'm seeing a very strange result from evaluation !   
Upon running (essentially) 
bazel-bin/inception/imagenet_eval --checkpoint_dir=/tmp/imagenet_train --eval_dir=/tmp/imagenet_eval,
Output for two consecutive batches is (essentially)

2016-06-07 22:32:50.391206: precision @ 1 = 0.0   (recall @ 5 also 0.0 )

I've seen this result on both: ImageNet data as well as a new dataset that I formatted via build_image_data.py.    Any idea why this may be happening ? 

I should note that the output being provided by the training script looks pretty reasonable:

2016-06-10 14:25:40.100805: step 890, loss = 7.11 (0.4 examples/sec; 180.106 sec/batch)
2016-06-10 14:55:30.076015: step 900, loss = 6.67 (0.4 examples/sec; 180.030 sec/batch)
2016-06-10 15:25:15.396038: step 910, loss = 6.90 (0.5 examples/sec; 123.756 sec/batch)
...
...

**Comment by julialintern** [comment]
I realize what my issue was :  Regarding the new dataset that I formatted via build_image_data.py:  I didnt realize that I needed to revise inception_data.py to reflect the correct number of classes (& samples) ..   Perhaps this could be added to the README ?
